### ISSUE TITLE
JS Example for Alerts & PopUps does not work

### ISSUE THREAD

**Issue by boris779** [SeleniumHQ/seleniumhq.github.io#565]
## 🐛 Bug Report

While translating to German, I noticed that on the js_alerts_prompts_and_confirmations page there is an example which should show an JavaScript Alert. Due to hugo this code is not functional

## To Reproduce

Go to https://www.selenium.dev/documentation/en/webdriver/js_alerts_prompts_and_confirmations/ 
Try to click on 'See an example Alert' which is not linked, and does not show an alert.

## Expected behavior

JavaScript Alert should pop up

## Environment
not relevant

**Comment by boris779** [comment]
I also tried to fix that issue.

I figured it out to add a custom js to the hugo template, that works and I prepared a pull request. But I didn't got a button or link working on that page. I tried some markups with {{% %}} but this is always marked by hugo with the "raw HTML omitted" comment in the created html files.

There is an option to allow "unsafe" HTML, but I do not think that this is the best solution.
